test/lib/rules/prevent-unused-connect-functions-test.js

```diff
@@ -0,0 +1,59 @@
+const rule = require('../../../lib/rules/prevent-unused-connect-functions');
+const RuleTester = require('eslint').RuleTester;
+
+const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2018, sourceType: 'module' } });
+
+ruleTester.run('prevent-unused-connect-functions', rule, {
+  valid: [
+    {
+      code: `export default connect(
+              (state) => ({}),
```

Comment:
It would be nice if we could mark this one as invalid, actually, since it's not connecting anything.

Reply:
Another rule should catch that since state is not used.

Reply:
Would that other rule catch this?

```js
export default connect(
  (_) => ({}),
  (_) => ({}),
  null,
  { withRef: true }
)(Foo);
```

I mostly ask because seeing "unused variable" or whatever would probably make me do that ^ or `_state`

Reply:
The eslint rule `no_unused_vars` still catches this case.